### PR TITLE
Known limitations

### PR DESCRIPTION
**Pull request by jmahaney**

### CODE REVIEW

File: Managed Services/getting-started-with-managed-wordpress.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  "contentIsHTML": false
+}}}
+
+**Overview**
+
+[WordPress](http://www.wordpress.org "WordPress.org") is a free open-source blogging tool and content management system (CMS) based on PHP and MySQL.
+
+CenturyLink Clouds solution is an Enterprise class, cloud-based solution for enterprise level customers.
+
+blah blah blah...........................
```
Comment (osbornm) [inline Managed Services/getting-started-with-managed-wordpress.md]: the master branch is meant to be able to be deployed to production at any point so no "placeholder" text can make it into the master branch

File: Managed Services/wordpress-known-limitations.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+* [__PHP mail__](#mail)
+
+---
+### <a name="multisite"></a>WordPress Multisite
+WordPress Multisite allows a single WordPress installation to house a network of separate sites. This feature is *disabled* by default in CenturyLink Cloud WordPress, and __is not__ supported.
+
+### <a name="storage"></a>Local File Storage
+CenturyLink Cloud WordPress __does not__ offer long term local file system storage for media, plugins, or themes. Traditional WordPress hosting on physical or virtual machines save site content to the local disk by default, but in the Cloud environment we have configured your WordPress site to work differently.
+
+> #### Media Content
```
Comment (osbornm) [inline Managed Services/wordpress-known-limitations.md]: this is a block comment I assume you didn't mean that?

File: Managed Services/wordpress-plugin-installation.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+  "author": "Bill Burge",
+  "attachments": [],
+  "contentIsHTML": false
+}}}
+
+CenturyLink Cloud WordPress sites use [GitHub](http://www.github.com "GitHub Homepage") to deploy new WordPress code including new plugins.
+
+In order to activate a plugin you will need to download a WordPress Plugin and upload it into the Master branch of the GitHub repository provided to you during your CenturyLink Cloud Wordpress site creation. This will then force a refresh of your WordPress blog and allow you to activate and configure the plugin.
+
+---
```
Comment (osbornm) [inline Managed Services/wordpress-plugin-installation.md]: please don't use horizontal rules.

File: Managed Services/wordpress-site-migration-to-centurylink-cloud.md [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+
+![](../images/wp_site_migration/wp_site_migration_15.png) 
+
**7. In the permissions column select READ (permission changes will then propagate through the directory structure)** 
+
+![](../images/wp_site_migration/wp_site_migration_16.png) 
+
+**8. Login to phpMyAdmin**
+
**9. Select the postmeta table**
```
Comment (osbornm) [inline Managed Services/wordpress-site-migration-to-centurylink-cloud.md]: there are two 1's in a row here

**Comment by osbornm** [comment]
In general I do not believe that all numbered lists should be bolded the styling of the site will handle making each row stand out without the need for the bolding.

**Comment by osbornm** [comment]
Also I recommend looking at the files in preview mode on github and making sure everything looks correct(ish) I noticed a few things like the block comments this way

**Comment by jmahaney** [comment]
@osbornm this was PR was meant to be requested against a fork of this repo that we keep as a staging area. Thank you for reviewing this, I'm going to close this PR. Thanks for the feedback though we will update before making a PR back into master.

**Pull request closed without merging**